Makefile

```
@@ -4,8 +4,17 @@ unit:
 integration:
 	node_modules/.bin/cucumber-js --tags "@algod or @assets or @auction or @kmd or @send or @indexer or @rekey_v1 or @send.keyregtxn or @dryrun or @compile or @applications or @indexer.applications or @applications.verified or @indexer.231 or @abi or @c2c or @compile.sourcemap" tests/cucumber/features --require-module ts-node/register --require tests/cucumber/steps/index.js
```

@algochoi Based on https://github.com/algorand/go-algorand-sdk/pull/360#discussion_r949337599, I think `@indexer.231` ought to be removed.

Note - I didn't assess which additional steps (if any) can also be removed.

```suggestion
	node_modules/.bin/cucumber-js --tags "@algod or @assets or @auction or @kmd or @send or @indexer or @rekey_v1 or @send.keyregtxn or @dryrun or @compile or @applications or @indexer.applications or @applications.verified or @abi or @c2c or @compile.sourcemap" tests/cucumber/features --require-module ts-node/register --require tests/cucumber/steps/index.js
```

---

I recommend resolving this in favor of the child PR #625

---

@tzaffi Thanks for the child PR - resolving!